src/libcore/cmp.rs

```diff
@@ -1,27 +1,22 @@
 //! Functionality for ordering and comparison.
 //!
-//! This module defines both [`PartialOrd`] and [`PartialEq`] traits which are used
-//! by the compiler to implement comparison operators. Rust programs may
-//! implement [`PartialOrd`] to overload the `<`, `<=`, `>`, and `>=` operators,
-//! and may implement [`PartialEq`] to overload the `==` and `!=` operators.
+//! This module contains various tools for ordering and comparing values. In
+//! summary:
 //!
-//! [`PartialOrd`]: trait.PartialOrd.html
-//! [`PartialEq`]: trait.PartialEq.html
+//! * [`Ord`] and [`PartialOrd`] are traits that allow you to define total and
+//!   partial orderings between values, respectively. They also let you overload
+//!   the `<`, `<=`, `>`, and `>=` operators.
+//! * [`Ordering`][cmp::Ordering] is an enum that's the return values of the
+//!   main functions of [`Ord`] and [`PartialOrd`], and describes an ordering.
+//! * [`Reverse`][cmp::Reverse] is a struct that allows you to easily reverse
+//!   an ordering.
+//! * [`Eq`] and [`PartialEq`] are traits that allow you to define total and
```

Comment:
I would move this bullet to the top since it's more important than the other things.

Reply:
hm, interesting. i did it this way because i see `Ord` as more important. that said, it might be better anyway because then the two are next to each other; this is the second most important thing to me, but it ended up further down in the list since reverse/ordering are related to ord

Reply:
Yes I think having them next to each other is a good idea / motivation :)